release/version

```diff
@@ -1 +1 @@
-v1.3.0
```

Comment:
Please update metadata.yaml to reflect the latest v1.4.0, without which clusterctl will not work.

Reply:
Updated